src/lib/components/Widget.js

```diff
@@ -250,7 +253,10 @@ export const Widget = React.forwardRef((props, forwardedRef) => {
 
   const widget = element !== null && element !== undefined ? (
     <ErrorBoundary
-      FallbackComponent={ErrorFallback}
+      FallbackComponent={() => {
+        near.config.errorCallback({scope: ErrorScopes.Boundary, message: src});
+        return <ErrorFallback/>;
+      }}
```

Comment:
`ErrorFallback` actually has props `{ error }` given to it. So we should pass through all props: https://github.com/NearSocial/VM/blob/fe647fe95bc68b1459f85310679cdb77389cfd32/src/lib/data/utils.js#L24

May want to report it as well.